src/login/logind-dbus.c

```diff
@@ -369,6 +369,31 @@ static int property_get_scheduled_shutdown(
         return sd_bus_message_close_container(reply);
 }
 
+static int property_get_maintenance_window(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        _cleanup_free_ char *s = NULL;
+        Manager *m = userdata;
+
+        assert(bus);
+        assert(reply);
+        assert(m);
+
+        if (m->maintenance_window) {
+                int r = calendar_spec_to_string(m->maintenance_window, &s);
```

Comment:
in our codebase `r` is the one local variable which we always use for storing error-code return values. As such it gets the special privilege to always be declared on the outermost {} block of a function, even if only needed in an inner {}.

Hence, please move `int r;` to the outer {}